lib/middlewares/filtered_cors.rb

```diff
@@ -1,5 +1,9 @@
+# frozen_string_literal: true
+
+# This class should eventually be replaced and its specs refactored
+# in favor of configuration based on the rack-cors gem
```

Comment:
This isn't a change worthy of a ticket, but I'm leaving the comment as a signpost that there's an opportunity for refactoring if we ever need to make deeper CORS changes.